### DISCUSSION TITLE
Difference in output XML and expected XML when using schema.

### DISCUSSION THREAD

**Discussion by abhishekvishnoi** [atlasmap/atlasmap#3489] [Q&A]
While transforming from a json to SOAP request .  I'm facing the following issue :
1. The actual output contains the namespace in the tags itself . 
2. However the expected output for request is having the namespaces as prefixes .

How can i achieve this , is there any option which can help me generate the expected output. 
[soap_namespace_issue_with_xml_schema.zip](https://github.com/atlasmap/atlasmap/files/7552082/soap_namespace_issue_with_xml_schema.zip)

![Screen_Capture_-_16_Nov__1_04_pm](https://user-images.githubusercontent.com/50903850/142141972-8359500c-5700-4543-bfa1-2dd79273c878.png)

**Comment by igarashitm** [comment]
There's no control for that. Since these are semantically same XML, we don't take care of the difference. The rule of thumb is we support/will support only what could be explicitly declared in XML schema. As you cannot distinguish these through XML schema, we won't support it.
